docs/UserGuide.md

```diff
@@ -301,13 +302,27 @@ Examples:
 * `re_edit 1 1 d/Tuesday`
 * `re_edit 1 1 t/14:00`
 
+### Export a person: `export`
+Export a person's details from the address book. 
+
+Format: `export [INDEX_OF_PERSON]`
+
+* Exports a person's details with the specified `INDEX`
```

Comment:
`INDEX` can be changed into `INDEX_OF_PERSON` (following the format) to make it clearer